lib/FusionInventory/Agent/Task/Inventory/Generic/Rudder.pm

```diff
@@ -7,6 +7,8 @@ use English qw(-no_match_vars);
 
 use FusionInventory::Agent::Tools;
 
+use JSON::PP;
```

Comment:
Even if JSON::PP is still necesarry for some FusionInventory agent modules, I would advise you to better depend on `UNIVERSAL::require` and add a `JSON::PP->require()` later in the related-eval. This will delay the load until it is really needed and even will save some memory usage while no custom script is found.

Reply:
i mimicked what was in Inventory/Virtualization/Docker.pm and HTTP/Client/Fusion.pm
i'll try the proposed solution